docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
Probably worth discussing sphinx in more detail tbh - personally still not convinced

Reply:
Do you want to do a in-person call? We could meet up via Discord e.g...

Reply:
Don't mind tbh. One outcome imo is we keep it for the merge, I create a branch. And then revert on main whilst we discuss. A large part of my issue is relying on github specific stuff as that goes against the whole distributed nature of git

Reply:
Hmm, that could make sense. What if we do both?

Reply:
Yeah sounds great, that way at least we can be unblocked :)